src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -205,6 +205,8 @@
 
     public static final String VERSION_CONTROL_URL = "versionControlUrl";
 
+    public static final String HEAD = "HEAD";
```

Comment:
I suggest reusing the constant defined in JGit for HEAD, see org.eclipse.jgit.lib.Constants.HEAD